conversions/conversions_test.go

```diff
@@ -115,6 +115,53 @@ func TestConvertLength(t *testing.T) {
 
 }
 
+func TestConvertArea(t *testing.T) {
+	a, err := ConvertArea(1000.0, constants.UnitMetres, constants.UnitKilometers)
+	assert.Equal(t, err, nil)
+	assert.Equal(t, a, 0.001)
+
```

Comment:
Since there are two kinds of constants (`UnitKilometers` and `UnitKilometres` etc) is it possible to add tests for them as well?